src/oaklib/datamodels/mapping_rules_datamodel.yaml

```diff
@@ -74,61 +80,43 @@ classes:
     attributes:
       subject_source_one_of:
         multivalued: true
+        description: The source of the subject to be matched.
+          Multiple values can be provided, it must match at least one.
```

Comment:
reading only from the gh diff, an example value looks like it would be useful to add